src/esp/physics/bullet/BulletRigidObject.h

```diff
@@ -442,6 +408,38 @@ class BulletRigidObject : public RigidObject,
   const Magnum::Range3D getCollisionShapeAabb() const;
 
  protected:
+  /**
+   * @brief Initializes this @ref BulletRigidObject as static scene geometry.
+   * See @ref PhysicsManager::staticSceneObject_. Sets @ref rigidObjectType_ to
+   * @ref RigidObjectType::SCENE. See @ref btCollisionObject.
+   * @param physicsSceneAttributes The template structure defining relevant
+   * phyiscal parameters for the physical scene.
+   * @param meshGroup The collision mesh data for the scene.
+   * @param bWorld The @ref btMultiBodyDynamicsWorld to which the scene should
+   * belong.
+   * @return true if initialized successfully, false otherwise.
+   */
+  bool initializeSceneFinalize(
```

Comment:
Assuming those are meant to be ever called from `initializeScene()`, this (and the "parent" implementation as well) could be `private` I think.

I'm doing a similar thing for plugin interfaces in magnum, the actual implementations being private to prevent them from being called by accident from outside (see e.g. [Trade::AbstractImporter](https://doc.magnum.graphics/magnum/classMagnum_1_1Trade_1_1AbstractImporter.html#pri-methods)).

Reply:
Thanks for suggestion here - I was thinking access was restricted from base class unless they were protected (Which is what i made them).

Reply:
Just FYI I still see them `protected`, not `private`. Did you actually make the change? :)

Reply:
oops.  will address this in next pr, along with ResourceManager ptr->ref conversion